wireless_modules/middle_module/boot.py

```diff
@@ -0,0 +1,58 @@
+# Author: Kunj
+# Last updated: 1/08/2020
+
+# Import relevant libraries
+import esp
+import network # To connect to the WIFI
+import config # File in the same path containing WIFI credentials and Pin to in built blue LED
+import time
+
+
+def do_connect(essid, password):
+    """
+    Tries to connect to the WIFI 5 times at max
+    :param essid: The WIFI name to connect to, ensure it's not in double quotes but single ones
+    :param password: The WIFI password to connect, ensure it's not in double quotes but single quotes
+    :return: None
+    """
+
+    # How many attempts to connect to the WIFI
+    attempts = 5
+
+    # Create station interface
+    station_interface = network.WLAN(network.STA_IF)
+
+    # Activate the interface
+    station_interface.active(True)
+
+    if station_interface.isconnected():
+        print('network config:', station_interface.ifconfig())
```

Comment:
Make sure that you consistently use either  `'` or `"` throughout the code where possible

Reply:
In the rest of our python code we use `"`, so for consistency, that would be prefered